bin/picca_fast_metal_dmat.py

```diff
@@ -419,7 +419,7 @@ def main():
             )
             userprint("Please specify the directory with option --delta-dir")
             sys.exit(1)
-    blinding = io.read_blinding(args.delta_dir)
+    blinding = io.read_blinding(args.in_dir, args.lambda_abs, args.lambda_abs2)
```

Comment:
@iprafols - you have changed here delta_dir for in_dir. Was this intended, or a copy-paste bug?

Reply:
ups, copy-paste bug, I'll fix it right away